agent.go

```diff
@@ -541,9 +542,9 @@ func (s *sandbox) listenToUdevEvents() {
 			}
 
 			s.Unlock()
-		} else if strings.HasPrefix(uEv.DevPath, sysfsMemOnlinePath) {
+		} else if strings.HasPrefix(sysfsDir+uEv.DevPath, sysfsMemOnlinePath) {
```

Comment:
You forgot to address the comment where I was saying that `filepath.Join()` is safer than `+`.

Reply:
ouch! updating now.